pkg/networkservice/connectioncontext/ipcontext/ipaddress/client.go

```diff
@@ -87,5 +88,8 @@ func (i *ipaddressClient) Request(ctx context.Context, request *networkservice.N
 }
 
 func (i *ipaddressClient) Close(ctx context.Context, conn *networkservice.Connection, opts ...grpc.CallOption) (*empty.Empty, error) {
+	if err := addDel(ctx, conn, i.vppConn, metadata.IsClient(i), false); err != nil {
+		log.FromContext(ctx).Error(err)
```

Comment:
I suppose that it is `Warn`, not `Error`, because nothing is actually failing.

Reply:
@Mixaster995 
please, use for `ipaddressServer` the same logic

Reply:
@glazychev-art  ok, done